### PR TITLE
chore(deps): update enzyme, use component api for react-intl test

### PR DESCRIPTION
**Pull request by alexkrolick**
- updates enzyme to add the wrappingComponent option
- use wrappingComponent to avoid manipulation of react-intl components outside the component tree

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -106,7 +106,6 @@
         "clearMocks": true,
         "restoreMocks": true,
         "moduleNameMapper": {
-            "react-intl": "<rootDir>/scripts/jest/mocks/react-intl-mocks.js",
```
Comment (priyajeet) [inline package.json]: line 47 exclusion for __mocks__

Comment (priyajeet) [reply package.json]: unless you want to move it under __tests__

Comment (alexkrolick) [reply package.json]: I'm removing that commit, that'll be on #1391

File: src/elements/content-sidebar/versions/__tests__/VersionsMenu-test.js [hunk 2]
```diff
@@ -1,15 +1,14 @@
 import * as React from 'react';
 import { IntlProvider } from 'react-intl';
-import { shallow } from 'enzyme/build';
 import messages from '../messages';
 import VersionsGroup from '../VersionsGroup';
-import { VersionsMenuComponent as VersionsMenu } from '../VersionsMenu';
```
Comment (jstoffan) [inline src/elements/content-sidebar/versions/__tests__/VersionsMenu-test.js]: The extra export in `VersionsMenu` can probably be deleted now.

**Pull request merged**